nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import rospy
+import actionlib
+import queue
+
+from actionlib_msgs.msg import GoalStatus
+from nightingale_msgs.msg import MissionPlanAction
+from nightingale_dispatcher.handoff_task import HandoffTask
+from nightingale_dispatcher.idle_task import IdleTask
+from nightingale_dispatcher.navigate_task import NavigateTask
+from nightingale_dispatcher.stock_task import StockTask
+from nightingale_dispatcher.task import Task
+from nightingale_dispatcher.triage_task import TriageTask
+
+class MissionPlanner:
+    def __init__(self):
+        rospy.init_node("mission_planner_node")
+
```

Comment:
Do we need other variables such as the robot state and a variable (context) for what branch the robot is traversing down? I think we need robot state for sure to help update the interface on what the robot is currently doing and we can also use it to help plan other actions.

Reply:
Probably want a listener to the estop topic to be able  to listen anytime and pause things